### PR TITLE
add roles to awx_collection

### PR DESCRIPTION
**Pull request by sean-m-sullivan**
##### SUMMARY
This is draft for merging the redhat_cop.controller_configuration collection into the awx.awx collection. One role is implemented in a single test file to serve as an easy to digest example of what would be the changes. 

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - awx collection

##### AWX VERSION
19.4.0

##### ADDITIONAL INFORMATION
Looking for feedback before myself and a few others take on the process of doing the full merge.

### CODE REVIEW

File: awx_collection/tests/integration/targets/project/tasks/run_it.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+- name: Run Integration Test
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  environment:
+      TOWER_HOST: https://tower-ansible-automation-platform.apps-crc.testing
+      TOWER_USERNAME: admin
+      TOWER_PASSWORD: jWidobmi5rfdw7yensgjM8RRMmkt2Pat
```
Comment (shanemcd) [inline awx_collection/tests/integration/targets/project/tasks/run_it.yml]: 👀

Comment (sean-m-sullivan) [reply awx_collection/tests/integration/targets/project/tasks/run_it.yml]: Thats temporary, let me remove it from the PR, I was leaving it in to run testing.

**Comment by IPvSean** [comment]
I love this idea, this expands on the core functionality of the modules into turn-key roles that can be used in the collection

**Comment by IPvSean** [comment]
@ffirg 🙏

File: awx_collection/roles/projects/README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+# controller_configuration.projects
+## Description
+An Ansible Role to create Projects on Ansible Controller.
+
+## Requirements
+ansible-galaxy collection install  -r tests/collections/requirements.yml to be installed
+Currently:
+  awx.awx
+  or
+  ansible.controller
```
Comment (AlanCoding) [inline awx_collection/roles/projects/README.md:10]: weird to list this as a requirement

Comment (sean-m-sullivan) [reply awx_collection/roles/projects/README.md:10]: I lifted and shifted the entire role, all the documentation is going to need to be rewritten to remove all references like this. Purpose of this draft is to start the conversation, definitely not to merge as is.

**Comment by john-westcott-iv** [comment]
@ffirg Do we want to consume this work from @sean-m-sullivan 🙏

**Comment by sean-m-sullivan** [comment]
We are on board for that.

**Comment by sean-m-sullivan** [comment]
closing in favor of #14118

**Pull request closed without merging**